From 6a7de145cf7e92436a706b9038fa02252516c992 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Fri, 7 Aug 2020 08:34:17 +0000
Subject: [PATCH 14/31] treewide: use standard intX types

__int64 is non-standard and is absent in musl libc.

References: KF-3361
---
 common/TimeUtil.cpp                    | 4 ++--
 common/include/kopano/platform.linux.h | 3 ++-
 2 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/common/TimeUtil.cpp b/common/TimeUtil.cpp
index da3075cc3..78d1ea284 100644
--- a/common/TimeUtil.cpp
+++ b/common/TimeUtil.cpp
@@ -99,9 +99,9 @@ time_t FileTimeToUnixTime(const FILETIME &ft)
 
 	if (sizeof(time_t) < 8) {
 		/* On 32-bit systems, we cap the values at MAXINT and MININT */
-		if (l < static_cast<__int64>(INT_MIN))
+		if (l < static_cast<int64_t>(INT_MIN))
 			l = INT_MIN;
-		if (l > static_cast<__int64>(INT_MAX))
+		if (l > static_cast<int64_t>(INT_MAX))
 			l = INT_MAX;
 	}
 	return l;
diff --git a/common/include/kopano/platform.linux.h b/common/include/kopano/platform.linux.h
index 7b49dd051..672e98235 100644
--- a/common/include/kopano/platform.linux.h
+++ b/common/include/kopano/platform.linux.h
@@ -13,6 +13,7 @@
  */
 #include <kopano/zcdefs.h>
 #include <vector>
+#include <cstdint>
 #include <cstdio>
 #include <cstdlib>
 #include <pthread.h>
@@ -74,7 +75,7 @@ typedef int	BOOL;
 typedef void* LPVOID;
 typedef char* LPSTR;
 typedef const char* LPCSTR;
-typedef __int64_t __int64;
+typedef int64_t __int64;
 
 #ifndef FALSE
   #define FALSE	0
-- 
2.29.2

